site2/website-next/contribute/document-preview.md

```diff
@@ -37,13 +37,21 @@ Follow these steps to preview documentation changes on the **master** branch.
 2. Run the following command to preview changes:
 
     ```bash
-    # Preview the next version
+    # Preview changes on latest stable version 
     sh start.sh
-    # Preview historical versions
-    sh start.sh [<version-number> ...]
+
+    # Preview changes on a specific version version
+    sh start.sh 2.10.x 
+
+    # Preview changes on multiple versions
+    sh start.sh 2.10.x 2.9.x ...
     ```
+    
+  By default, a browser window will open at http://localhost:3000 to show the changes.
 
-3. By default, a browser window will open at http://localhost:3000 to show the changes.
+  > **Tip**
+  >
+  > If you want to preview changes on `master`, change the URL to `http://localhost:3000/docs/next`.
```

Comment:
```suggestion
:::tip

If you want to preview changes on `master`, change the URL to `http://localhost:3000/docs/next`.

:::
```

Let's use admonitions consistently. Also IIRC the admonitions rendering will fail if leading idents exist.

Reply:
The original content seems also rendered wrongly:

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/18818196/206675530-05168131-5f83-45f0-892b-99c303bd4ad3.png">

Please preview locally :)

Reply:
Emmm...It seems a two-space indent can be rendered properly:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/18818196/206676022-e879dc28-509f-473a-8e1b-a28dafd89511.png">

But we may unindent for both this tip and the following screenshot?